src/GoogleMaps.webmodeler.ts

```diff
@@ -18,12 +17,12 @@ export class preview extends Component<GoogleMapContainerProps, {}> {
         if (!warnings) {
             reactElement = createElement(Map, this.transformProps(this.props));
         } else {
-            reactElement = DOM.div({},
-                createElement(Alert, { message: warnings }),
+            reactElement = createElement("div", {},
+                createElement(Alert, { bootstrapStyle: "danger", message: warnings }),
```

Comment:
Add appropriate class name: `widget-google-maps-alert`. Check with the core components.